bigbluebutton-html5/imports/ui/components/shortcut-help/component.jsx

```diff
@@ -115,6 +123,20 @@ const ShortcutHelpComponent = (props) => {
     </tr>
   ));
 
+  shortcutItems.push((
+    <tr key={_.uniqueId('hotkey-item-')}>
+      <td className={styles.keyCell}>Right Arrow</td>
+      <td className={styles.descCell}>{intl.formatMessage(intlMessages.nextSlideDesc)}</td>
+    </tr>
+  ));
+
+  shortcutItems.push((
+    <tr key={_.uniqueId('hotkey-item-')}>
+      <td className={styles.keyCell}>Left Arrow</td>
```

Comment:
These label aren't `intl`

Reply:
Should they be added to `intl` or treated like the Alt and Ctrl?